integration-cli/docker_cli_inspect_test.go

```diff
@@ -93,13 +88,10 @@ func (s *DockerSuite) TestInspectTypeFlagContainer(c *check.C) {
 
 	formatStr := fmt.Sprintf("--format='{{.State.Running}}'")
 	out, exitCode, err := dockerCmdWithError("inspect", "--type=container", formatStr, "busybox")
-	if exitCode != 0 || err != nil {
-		c.Fatalf("failed to inspect container: %s, %v", out, err)
-	}
+	c.Assert(err, checker.IsNil)
```

Comment:
@HuKeping @MHBauer if you use dockerCmd (source [here](https://github.com/docker/docker/blob/master/pkg/integration/dockerCmd_utils.go#L35)), the error will have some context :wink:.